src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -318,9 +332,13 @@ private boolean isOperationsUser(final String username){
 		return false;
 	}
 
-	private synchronized void syncAcl() {
-		logger.debug("Syncing the ACL to ElasticSearch");
+	private void syncAcl() {
```

Comment:
Doesn't `synchronized` mean that syncAcl can only be called by 1 thread at a time?  That is, why do you need to replace the `sychronized` method with an explicit lock?